packages/tools/src/stateManagement/segmentation/polySeg/polySegConverters.js

```diff
@@ -0,0 +1,387 @@
+import { expose } from 'comlink';
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
+import ICRPolySeg from '@icr/polyseg-wasm';
+import { utilities } from '@cornerstonejs/core';
+import { getBoundingBoxAroundShapeWorld } from '../../../utilities/boundingBox';
+import { pointInShapeCallback } from '../../../utilities';
+import { isPointInsidePolyline3D } from '../../../utilities/math/polyline';
+
+const obj = {
+  polySeg: null,
+  polySegInitializing: false,
+  polySegInitializingPromise: null,
+  async initializePolySeg(progressCallback) {
```

Comment:
Could these be static members of a class instead of object members in a typescript class?  Don't know if that works, but if it does it would be nice.

Reply:
It does not work last time i tried the worker stuff